docs/userGuide/syntax/panels.md

```diff
@@ -145,6 +145,17 @@ plain text ...
 </variable>
 </include>
 
+<div id = "script_and_styles_warning">
+<box type = "warning" header = "#### Global Effects of the Script and Styles from the Imported Externals">
+
+Importing external resources that contains `script` or `styles` can inadvertently take global effects on your MarkBind website. Due to hoisting during processing, imported scripts and stylesheets affects the entire page. This could potentially alter its appearance and behavior beyond the intended scope.
+
+For example, if a CSS file imported via such means styles headings to be red, this change will be reflected page-wide.
+
+To safeguard against unintended consequences, it's advisable to explore alternative strategies. Consider directly incorporating the code or customizing styles to target specific elements or classes not used universally. This approach grants more precise control over your website's presentation and reduces the risk of unexpected changes.
```

Comment:
> To safeguard against unintended consequences, it's advisable to explore alternative strategies. Consider directly incorporating the code or customizing styles to target specific elements or classes not used universally. This approach grants more precise control over your website's presentation and reduces the risk of unexpected changes.

To safeguard against unintended consequences, consider directly incorporating the code or customizing styles to target specific elements or classes not used universally. This approach grants more precise control over your website's presentation and reduces the risk of unexpected changes.

Reply:
Noted;
Got "it's advisable to explore alternative strategies." deleted.